### ISSUE TITLE
Add option to normalize at one point only

### ISSUE THREAD

**Issue by erwanp** [radis/radis#127]
Extension of previous PR :

--- 

(details)
I would make clearer the if ... then ... else here

```
 if len(wrange) == 2:
             blabla
 elif isinstance(wrange, (int, float)):
             blabla
  else:
             UserWarning

```

_Originally posted by @minouHub in https://github.com/radis/radis/pull/122#discussion_r473656658_

**Comment by 4the4ryushin** [comment]
is the issue still open?

**Comment by erwanp** [comment]
yes

**Comment by minouHub** [comment]
This issue was closed with #627
